ci-operator/step-registry/cucushift/installer/check/azure/confidential/trustedlaunch/cucushift-installer-check-azure-confidential-trustedlaunch-commands.sh

```diff
@@ -4,6 +4,12 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+# save the check status into install-post-check-status.txt
+# for junit xml file generated
+post_check_exit_code=0
+EXIT_CODE_POSTCHECK=100
+trap 'if [[ "$?" != 0 ]]; then post_check_exit_code=${EXIT_CODE_POSTCHECK}; fi; echo "$post_check_exit_code" > "${SHARED_DIR}/install-post-check-status.txt"' EXIT TERM
```

Comment:
can we directly use:
```
EXIT_CODE_POSTCHECK=100
trap 'if [[ "$?" != 0 ]]; then echo "$post_check_exit_code" > "${SHARED_DIR}/install-post-check-status.txt"; fi' EXIT TERM
```
to save `post_check_exit_code` here?

Reply:
If so, we just only capture failed jobs, and skip all passed jobs, because post-check junit file is generated when `install-post-check-status.txt` exists. 
Then in component readiness, I'm afraid that test case "install should succeed: post check" will be 100% failed if any job failed at post-check step is captured, is it right?

Reply:
Good point, if that, let us keep it.